JHUGenLexicon/src/JHUGenLexiconTranslator.cc

```diff
@@ -484,41 +722,75 @@ std::vector<std::vector<double>> JHUGenLexiconTranslator::getTranslationMatrix(
         res[coupl_efthbasis_tCza][coupl_eftjhu_ghzgs4] = -((2*sqrt(cw)*sqrt(sw))/pow(e,2));
         res[coupl_efthbasis_Caa][coupl_eftjhu_ghgsgs2] = -(2/pow(e,2));
         res[coupl_efthbasis_tCaa][coupl_eftjhu_ghgsgs4] = -(2/pow(e,2));
```

Comment:
Are these other lines also affected?

Reply:
(Referring to the change one line below.)

Reply:
Yes? I am unsure what you are referencing here.